modules/control/warps.lua

```diff
@@ -202,22 +198,15 @@ function Warps.remove_warp_area(warp_id)
     local position = warp.position
     local surface = warp.surface
     local radius = config.standard_proximity_radius
-    local radius2 = radius^2
 
     -- Check that a warp area was created previously
-    local base_tile = warp.old_tile
-    if not base_tile then return end
+    local old_tile = warp.old_tile
+    if not old_tile then return end
 
-    -- Reset all the tiles that were replaced
+    -- Loop over warp tiles and set them to the old tile that was below
```

Comment:
Unsure what exactly this is doing, but would something like this be better
```suggestion
    -- Restore the original tiles before the creation of the warp
```